android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java

```diff
@@ -444,6 +448,36 @@ public void onError(String oldUrl) {
     }
   }
 
+  private void warnUserIfGooglePwsWouldFilterUrl(String url){
+    if (Utils.isGoogleApiKeyAvailable(getActivity())){
+      PwsClient pwsClient = new PwsClient();
+      Utils.setPwsEndPointToGoogle(getActivity(), pwsClient);
+      pwsClient.resolve(Arrays.asList(url), new PwsResultCallback() {
+        public void showToastOnUiThread(final int messageId){
+          getActivity().runOnUiThread(new Runnable(){
+            @Override
+            public void run() {
+              Toast.makeText(getActivity(), messageId, Toast.LENGTH_LONG).show();
+            }
+          });
+        }
+
+        @Override
+        public void onPwsResult(PwsResult pwsResult) {
+        }
+
+        @Override
+        public void onPwsResultError(Collection<String> urls, int httpResponseCode, Exception e) {
+          if (urls.iterator().next().contains("https")){
```

Comment:
We should toLower the URL before checking the scheme so we can match (eg) HTTPS://www.google.com